src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Parameterize this service and add its implementation in a separate class.
  */
-public interface MachineService {
+public interface MachineService<T> {
```

Comment:
```suggestion
public interface MachineService<T extends Machine> {
```
Якщо так напишемо, то убезпечемо ті кейси, коли хтось захоче написати наприклад ... implements MachineService<String>